packages/lib/factorio/server.js

```diff
@@ -791,6 +791,7 @@ class FactorioServer extends events.EventEmitter {
 
 		try {
 			let [code, signal] = await events.once(this._server, "exit");
+			void signal;
```

Comment:
I don't understand what `void` is supposed to do here? I can't find a reference to it in the lgtm link either.

Reply:
It's a common way to mark unused parameters or variables as intentionally not used.